lib/linguist/languages.yml

```diff
@@ -1596,6 +1596,21 @@ Fortran Free Form:
   codemirror_mode: fortran
   codemirror_mime_type: text/x-fortran
   language_id: 761352333
+FreeBasic:
+  type: programming
+  color: "#867db1"
+  extensions:
+  - ".bi"
+  - ".bas"
+  - ".frm"
+  - ".inc" 
```

Comment:
`.inc` is a very commonly-used file extension, so we'll need a heuristic for matching FormBasic `.inc` files.